common/src/main/java/com/datastrato/gravitino/json/JsonUtils.java

```diff
@@ -236,9 +237,14 @@ private static class ObjectMapperHolder {
   /**
    * Get the shared ObjectMapper instance for JSON serialization/deserialization.
    *
-   * @return The ObjectMapper instance.
+   * <p>This instance should only be used for testing. For production or other environments, obtain
+   * an {@link ObjectMapper} from the appropriate {@code ObjectMapperProvider} on the server or
+   * client side.
+   *
+   * @return the shared {@link ObjectMapper} instance for tests.
    */
-  public static ObjectMapper objectMapper() {
+  @VisibleForTesting
+  public static ObjectMapper objectMapperForTest() {
```

Comment:
If it's only for testing,  `@VisibleForTesting` seems to be enough, the name `objectMapperForTest` is redundant.

Reply:
ok

Reply:
done